mods/cnc/maps/shellmap/shellmap.lua

```diff
@@ -1,3 +1,362 @@
+CommandoTeam = {"rmbo", "rmbo"}
+GDIAttackTeam = {"e1", "e1", "e2"}
+GDIInfantryTeam = {"e1", "e1", "e1", "e2", "e2"}
+GDIVehicleTeam = {"jeep","jeep"}
+NodInfantry =
+	{
+		{{"e1", "e1", "e3"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e1", "e1", "e1", "e3", "e3"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e4", "e4"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"e1", "e1", "e1", "e1", "e3", "e3", "e4", "e4" },function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end}
+	}
+NodVehicles =
+	{
+		{{"ltnk", "ltnk", "ltnk"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"bike", "bike", "bike"},function(a)
+			a.AttackMove(WestSide.Location)
+			a.AttackMove(MCVDeploy.Location)
+		end},
+		{{"bike", "bike"},function(a)
+			a.AttackMove(HandBuild.Location+CVec.New(3,5))
+			a.AttackMove(MCVDeploy.Location)
+		 end},
+		{{"bggy", "bggy", "bggy"},function(a)
+			a.AttackMove(HandBuild.Location+CVec.New(3,5))
+			a.AttackMove(MCVDeploy.Location)
+		 end}
+	}
+OutpostInfantry =
+	{
+		{{"e1", "e1", "e3"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e1", "e1", "e1", "e3", "e3"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e4", "e4"}, function(a) a.AttackMove(MCVDeploy.Location) end },
+		{{"e1", "e1", "e1", "e1", "e3", "e3", "e4", "e4" }, function(a) a.AttackMove(MCVDeploy.Location) end }
+	}
+
+LSTREntryPath = {LSTRStart.Location, LSTRMove.Location, LSTRLand.Location}
+LSTMEntryPath = {LSTMStart.Location, LSTMMove.Location, LSTMLand.Location}
+LSTLEntryPath = {LSTLStart.Location, LSTLMove.Location, LSTLLand.Location}
+
+LSTRExitPath = {CPos.New(LSTRMove.Location.X,LSTLStart.Location.Y), CPos.New(8,LSTLStart.Location.Y)}
+LSTMExitPath = {LSTMMove.Location, CPos.New(8,LSTMStart.Location.Y)}
+LSTLExitPath = {CPos.New(LSTLMove.Location.X,LSTRStart.Location.Y), CPos.New(8,LSTRStart.Location.Y)}
+
+--Repeated Functions START
+UnloadLST = function(lst,cargo)
```

Comment:
This was necessary to get the proper exit direction out of the units. Otherwise, they exit Northeast unlike they should.